facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -143,6 +144,81 @@ <h2 class="container-title @if(collectionIndex > 0) { container-title--not-first
     }
 }
 
+@renderMarquee(marquee: LiveIntentMarquee) = {
+    <table align="center" border="0" cellpadding="0" cellspacing="0" style="margin-left:auto; margin-right:auto;">
+        <tr>
+            <td colspan="2">
+                <a href="http://li.theguardian.com/click?s=@{marquee.ids._1}&layout=marquee&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%" rel="nofollow">
+                    <img border="0" src="http://li.theguardian.com/imp?s=@{marquee.ids._1}&layout=marquee&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%" style="display: block; width:100%; height:auto;" width="500">
+                </a>
+            </td>
+        </tr>
+        <tr style="display:block; height:1px; line-height:1px;">
+            <td><img height="1" src="http://li.theguardian.com/imp?s=@{marquee.ids._2}&sz=1x1&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%" width="10"></td>
+            <td><img height="1" src="http://li.theguardian.com/imp?s=@{marquee.ids._3}&sz=1x1&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%" width="10"></td>
+        </tr>
+        <tr>
+            <td align="left">
+                <a href="http://li.theguardian.com/click?s=@{marquee.ids._4}&sz=116x15&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%" rel="nofollow">
+                    <img border="0" src="http://li.theguardian.com/imp?s=@{marquee.ids._4}&sz=116x15&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%">
+                </a>
+            </td>
+            <td align="right">
+                <a href="http://li.theguardian.com/click?s=@{marquee.ids._5}&sz=69x15&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%" rel="nofollow">
+                    <img border="0" src="http://li.theguardian.com/imp?s=@{marquee.ids._5}&sz=69x15&li=@{marquee.newsletterId}&m=%%=v(@@hashedEmail)=%%&p=%%jobid%%">
+                </a>
+            </td>
+        </tr>
+    </table>
+}
+
+@renderMPU(mpu: LiveIntentMPU) = {
+    <table border="0" cellpadding="0" cellspacing="0">
```

Comment:
The email framework we use applies some very general styles, e.g. to all `<td>`. We would need to make sure these didn't have a detrimental effect on the appearance of the ads, or scope them somehow so they don't apply to the ads.

This is one of many things that are convincing me that we need to hack the framework itself rather than trying to use it as a library. As part of my upgrade work I've switched the framework to be one of our dependencies in our `package.json`, thinking that this would make things easy if we wanted to incorporate updates to the framework in future. But I'm becoming more and more disillusioned with what it actually provides us. I'm thinking a better option could be to just add the Sass files as part of our repo and then we can modify them to make them fit for purpose. First thing to do would be to switch all selectors to use classes. This would fix the issue here, and make it more likely that we could do without style inlining for Gmail